src/AsyncTCP.cpp

```diff
@@ -31,6 +31,9 @@ extern "C"{
 }
 #include "esp_task_wdt.h"
 
+#define CONFIG_ASYNC_TCP_STACK 2*8192
+
+#define CONFIG_ASYNC_TCP_PRIORITY 10
```

Comment:
Task priority = 10 seems to be very brutal, considering that priority of the "-arduino task is 1, idle task has 0, and there are realtime applications "out there" with a very well-designed task priority structure to ensure that each task can achieve its relit e response times. I'd suggest to use priority 1 or 2, to better support realtime applications where delivering web pages is not at highest priority.